NEWS.md

```diff
@@ -1,5 +1,9 @@
 # testthat 0.9.1.9000
 
+* `test_dir`, `test_package`, and `test_check` have an added `...` argument
```

Comment:
Doesn't look like you actually modified `test_check()` (which is right) but can you remove from here?

Also house style is to follow function names with `()`, e.g.` test_dir()`.